src/plugins/controls/public/control_group/editor/control_editor.tsx

```diff
@@ -198,27 +198,27 @@ export const ControlEditor = ({
               />
             </EuiFormRow>
           )}
-          <EuiFormRow label={ControlGroupStrings.manageControl.getFieldTitle()}>
-            <FieldPicker
-              filterPredicate={(field: DataViewField) => {
-                return Boolean(fieldRegistry?.[field.name]);
-              }}
-              selectedFieldName={selectedField}
-              dataView={dataView}
-              onSelectField={(field) => {
-                onTypeEditorChange({
-                  fieldName: field.name,
-                });
-                const newDefaultTitle = field.displayName ?? field.name;
-                setDefaultTitle(newDefaultTitle);
-                setSelectedField(field.name);
-                if (!currentTitle || currentTitle === defaultTitle) {
-                  setCurrentTitle(newDefaultTitle);
-                  updateTitle(newDefaultTitle);
-                }
-              }}
-            />
-          </EuiFormRow>
+          {fieldRegistry && (
+            <EuiFormRow label={ControlGroupStrings.manageControl.getFieldTitle()}>
+              <FieldPicker
+                filterPredicate={(field: DataViewField) => Boolean(fieldRegistry?.[field.name])}
```

Comment:
**nit:** Since you added the `fieldRegistry && ...` catch, could this be changed to:
```suggestion
                filterPredicate={(field: DataViewField) => Boolean(fieldRegistry.[field.name])}
```

Reply:
Nice, committed this!